components/03-modules/footer/footer.hbs

```diff
@@ -1,5 +1,27 @@
 <footer class="footer">
-    <div class="container">
-        Footer
+    <div class="container footer__handler">
```

Comment:
Wouldn't it be better to use this container as a mixin and include it in classes that we need to act like containers ? Or just extend.. This is how customizing bootstrap works. What do you think ? It would allow us to use only one class here.

Reply:
ask other devs is it good idea and than create issue for this

Reply:
leave it that way